doc/introduction.rst

```diff
@@ -7,12 +7,81 @@ Introduction: nistats in a nutshell
     :depth: 1
 
 
-What is nistats: Bold-fMRI analysis
+What is nistats?
 ===========================================================================
 
-.. topic:: **Why use nistats?**
+.. topic:: **What is nistats?**
 
-    Nistats is a python module to perform voxel-wise univariate analyses of functional magnetic resonance images (fMRI), using general linear models. It provides functions to create design matrices, at the subject and group levels, to estimate them from images series and to compute statistical maps (contrasts)
+   Nistats is a python module to perform voxel-wise analyses of functional magnetic resonance images (fMRI) using linear models. It provides functions to create design matrices, at the subject and group levels, to estimate them from images series and to compute statistical maps (contrasts). It allows to perform the same statistical analyses as `a SPM`_ or `a FSL`_ (but it does not provide tools for preprocessing stages (realignement, spatial normalization, etc.); for this, see `a nipype`_.
+
+.. _a SPM: https://www.fil.ion.ucl.ac.uk/spm/
+
+.. _a FSL: https://www.fmrib.ox.ac.uk/fsl
+
+.. _a nipype: https://nipype.readthedocs.io/en/latest/>
```

Comment:
The link should be https://nipype.readthedocs.io/en/latest/ not https://nipype.readthedocs.io/en/latest/>